src/idl/include/idl/tree.h

```diff
@@ -397,6 +399,23 @@ struct idl_enum {
   idl_node_t node;
   struct idl_name *name;
   idl_enumerator_t *enumerators;
+  uint16_t bit_bound;
+  idl_extensibility_t extensibility;
+};
+
+typedef struct idl_bit_value idl_bit_value_t;
+struct idl_bit_value {
+  idl_node_t node;
+  struct idl_name *name;
+  uint16_t position;
```

Comment:
Probably the same issue as with the bit_bound member? Normally this would be initialized to `0`, but if `@position` is used and we provide two of the same values, that'd go unnoticed.

Reply:
See above, I've added some additional checks